subworkflows/local/make_chunks/make_chunks.nf

```diff
@@ -0,0 +1,67 @@
+include { BCFTOOLS_INDEX                    } from '../../../modules/nf-core/bcftools/index/main'
+include { BCFTOOLS_INDEX as BCFTOOLS_INDEX_2} from '../../../modules/nf-core/bcftools/index/main'
+include { BCFTOOLS_INDEX as BCFTOOLS_INDEX_3} from '../../../modules/nf-core/bcftools/index/main'
+include { GLIMPSE_CHUNK                     } from '../../../modules/nf-core/glimpse/chunk/main'
+include { BCFTOOLS_CONVERT                  } from '../../../modules/nf-core/bcftools/convert/main'
+include { BCFTOOLS_NORM                     } from '../../../modules/nf-core/bcftools/norm/main'
+include { BCFTOOLS_VIEW                     } from '../../../modules/nf-core/bcftools/view/main'
+
+
+workflow MAKE_CHUNKS {
+
+    take:
+    ch_reference                           // channel: [ val(meta),vcf ]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    // Make chunks
+    ch_vcf_csi_chr = ch_reference.map{meta, vcf, csi -> [meta, vcf, csi, meta.chr]}
+    GLIMPSE_CHUNK(ch_vcf_csi_chr)
+
+    // Rearrange chunks into channel
+    ch_chunks = GLIMPSE_CHUNK.out.chunk_chr
+                    .splitText()
+                    .map { metamap, line ->
+                        def fields = line.split("\t")
+                        def startEnd = fields[2].split(':')[1].split('-')
+                        [metamap, metamap.chr, startEnd[0], startEnd[1]]
+                    }
+
+    ch_fasta = Channel.value([file(params.fasta,checkIfExists: true)])
```

Comment:
The channel fasta is already available in the previous workflow.
You can simply add it to your input parameter.
It's in the form [meta, fasta, fai]